docs/install.rst

```diff
@@ -191,7 +191,7 @@ The further dependencies provide more specific features:
 - `mpmath <http://mpmath.org/>`_: Used for the 'kraft-burrows-nousek'
   interval in `~astropy.stats.poisson_conf_interval`.
 
-- `asdf <https://github.com/spacetelescope/asdf>`_ |minimum_asdf_version| or later: Enables the
+- `asdf-astropy <https://github.com/astropy/asdf-astropy>`_: Enables the
```

Comment:
Hold on, I thought asdf-astropy dependency on top of asdf, not replacing it?

Also if there is a minversion for asdf-astropy we need to pin here, we have to declare it on this page too.

Reply:
asdf-astropy has its own asdf dependency, and I think we'd better not have dueling asdf dependencies between astropy and asdf-astropy?

Reply:
OK, in that case, can we set a minversion of asdf-astropy ?

Reply:
(reverted this and restored asdf to the `[all]` requirements)